src/analytics.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable eslint-comments/no-unlimited-disable */
+/* eslint-disable */
+if (window.SGJavascriptBridge) { // TODO: check for iOS and do the polling
+  var shopgateShopNumber = 'REPLACE_WITH_SHOP_NUMBER';
+
+  var scriptFileName = window.location.pathname.substr(1);
+  if (scriptFileName === 'checkout/order-confirmation' || scriptFileName === 'finishorder.php') {
+
+    (function(src) {
+      var s, r, t;
+      r = false;
+      s = document.createElement('script');
+      s.type = 'text/javascript';
+      s.src = src;
+      t = document.getElementsByTagName('script')[0];
+      t.parentNode.insertBefore(s, t);
+      window.__shopgate_aq = window.__shopgate_aq || [];
+      window.sgAnalytics = function() { window.__shopgate_aq.push(arguments); };
+    })
+    ('https://data.shopgate.com/tracking/shopgate-analytics-sdk.production.min.js');
+
+    function ShopgateUniversalTracking() {
+      this._addTrans = addTrans;
```

Comment:
As we just had a short discussion on this topic this is fine, we should just document it in the readme file.

Reply:
ok its because BigCommerce is checking in their JavaScript if window.pageTracker is set and if so BigCommerce will call the three methods that are seems to be unused